src/logger/index.js

```diff
@@ -86,5 +86,9 @@ module.exports = ( namespace, { transport, cluster } ) => {
 		level,
 	} );
 
+	process.on( 'uncaughtException', err => {
```

Comment:
Would this mean that we'd have a separate `uncaughtException` handler for each logger instance? If so, that may not be ideal.

Reply:
oh yes it is, this should move outside

Reply:
Fixed here https://github.com/Automattic/vip-go-node/pull/66/commits/dbfe13267199a5010663ef7cec2cc86e4e4af468

Reply:
Two options I can think of:

1) Register the handlerin `logger.js` with a generic namespace (like `vip-go:uncaught-exception`).

or

2) Close this out and leave it to the applications to log and track `uncaughtException` events (could be part of the setup/integrations docs).

Reply:
For the first one, we need to be sure to give the user the possibility to pass their own transport and configuration. This will be a duplication as this is what we are already doing when creating a new logger.

I'm more with the second option and adding this explicitly in the docs

Reply:
Makes sense. Let's go with a docs approach then.

We'll also want to make sure this is in place for all our apps.